src/app.py

```diff
@@ -87,9 +87,19 @@ def bootstrap(_return=0):
                         logger.info('Load test is stopped.')
 
                         logger.info('Downloading reports...')
+                        time.sleep(4)
                         report_path = os.path.join(os.getcwd(), 'reports')
                         os.makedirs(report_path)
 
+                        for _url in ['requests', 'distribution']:
+                          res = requests.get(url=master_url + '/stats/' + _url)
+                          with open(os.path.join(report_path, _url + '.json'), "wb") as file:
+                            file.write(res.content)
+
+                          res = requests.get(url=master_url + '/stats/' + _url + '/csv')
```

Comment:
this line does the similar things if we compare with previous line, could we can simplify it?

Reply:
IMO it's more effort that value, but you can do that if you wish